lxd/api_project.go

```diff
@@ -1032,7 +1032,9 @@ func projectNodeConfigDelete(d *Daemon, s *state.State, name string) error {
 		}
 
 		// Unmount the project-specific storage volumes.
-		for _, projectStorageVolume := range []string{config.StorageImagesVolume(name), config.StorageBackupsVolume(name)} {
+		localConfig := s.LocalConfig.Dump()
```

Comment:
@skozina what is the difference between `node.ConfigLoad` above (in the 1config` var) and `s.LocalConfig.Dump()` in the `localConfig` var? They both seem to be containing local node config.